ngraph/test/type_prop/experimental_detectron_detection_output.cpp

```diff
@@ -41,6 +41,25 @@ TEST(type_prop, detectron_detection_output)
     EXPECT_EQ(detection->get_output_shape(0), (Shape{rois_num, 4}));
     EXPECT_EQ(detection->get_output_shape(1), (Shape{rois_num}));
     EXPECT_EQ(detection->get_output_shape(2), (Shape{rois_num}));
+
+
+    rois = std::make_shared<op::Parameter>(element::f32, PartialShape::dynamic(2));
```

Comment:
Can we add more tests to check how it works if deltas is static and scores is dynamic, for example?